tests/jquery.flot.logaxis.Test.js

```diff
@@ -262,15 +262,15 @@ describe("integration tests for log scale functions", function() {
         expect(axes.yaxis.min).toBe(0.0001);
     });
 
-    it('should extend PREFERRED_LOG_TICK_VALUES if too little interval', function() {
+    it('should use EXTENDED_LOG_TICK_VALUES if the interval is too little', function() {
         var logdata = [
-            [[0, 1E40], [1, 4E40], [2, 8E40], [3, 2E41], [4, 6E41], [5, 9E41], [6, 3E42]],
-            [[0, 1E38], [1, 4E39], [2, 8E40], [3, 2E41]],
+            [[0, 1E40], [1, 4E40], [2, 8E40], [3, 1E41], [4, 2E41], [5, 4E41]],
+            [[0, 1000], [1, 1010], [2, 1E4], [3, 1E5]],
             [[0, 1], [1, 3], [2, 10], [3, 30], [4, 40], [5, 50]]
         ],
             expectedTicks = [
-            ['1e40', '5e40', '1e41', '5e41', '10e41'],
-            [ '1e38', '5e38', '10e38', '5e39', '1e40', '5e40', '1e41'],
+            ['1e40', '5e40', '1e41'],
+            ['1000', '5000', '10000', '50000', '1e5'],
```

Comment:
I think we are switching to "e mode" way too soon. I think we can reasonably display up to 10 million (9.999.999) in decimal mode.